### DISCUSSION TITLE
Can i rename my project app name?

### DISCUSSION THREAD

**Discussion by niklasgrewe** [blitz-js/blitz#2315] [Q&A] [answered]
Hi, to create a new blitz app, i am using the cli command:

```
blitz new myAppName
```

I have seen that this name is not used for the project folder, but also for the cookies etc. Can I change this name later? If so, how would I have to do that or where would I have to change the app name everywhere?

**Comment by flybayer** [comment, marked as answer]
That name is only used in the readme, package.json name field, and cookies. Currently just change the package.json field. But after https://github.com/blitz-js/blitz/pull/2311 is released, then you can set the cookie name separate from package.json
